config.yml

```diff
@@ -0,0 +1,35 @@
+defaults:
+  hostname:           https://events.taskcluster.net
+  port:               undefined
+  forceSSL:           false
+  trustProxy:         false
+  influx:
+    maxDelay:         5*60
+    maxPendingPoints: 250
+  events:
+    statsComponent:   events
+
+production: # profile 'production'
+  hostname:           !env HOSTNAME
+  port:               !env:number PORT
+  pulse:
+    username:         !env PULSE_USERNAME
+    password:         !env PULSE_PASSWORD
+  taskcluster:
+    credentials:
+      accessToken:    !env TASKCLUSTER_ACCESS_TOKEN
+      clientId:       !env TASKCLUSTER_CLIENT_ID
+  influx:
+    connectionString: !env:string INFLUX_DB_CONNECTION_STRING
+test: # profile 'test'
+  hostname:           http://localhost
+  port:               12345
+  forceSSL:           false
+  pulse:
+    username:         !env PULSE_USERNAME
```

Comment:
These aren't necessary. It will just use whatever was in defaults if left unspecified. Move the corresponding section in the production set into default and you'll be all set.